sdk/storage/storage-blob/src/ContainerClient.ts

```diff
@@ -733,11 +751,20 @@ export class ContainerClient extends StorageClient {
   }
 
   /**
-   * Creates a BlockBlobClient object.
+   * Creates a {@link BlockBlobClient}
    *
    * @param {string} blobName A block blob name
    * @returns {BlockBlobClient}
    * @memberof ContainerClient
+   *
+   * @example
+   * ```js
+   * const content = "Hello world!";
+   *
+   * const blobClient = containerClient.getBlobClient("<blob name>");
+   * const blockBlobClient = blobClient.getBlockBlobClient();
```

Comment:
Simplify to `const blockBlobClient = containerClient.getBlockBlobClient("<blob name>");`